ext/standard/tests/general_functions/phpinfo-header-anchors.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+phpinfo() with clickable anchor tags
+--DESCRIPTION--
+This tests for the HTML anchor tags presence in phpinfo() outputs.
+The --POST-- section is used to force using CGI SAPI instead of the
+default CLI SAPI.
+--POST--
+a=b
```

Comment:
You can use an empty [`--CGI--` section](http://qa.php.net/phpt_details.php#cgi_section) instead.

Reply:
I couldn't figure out why, by using `--CGI--` [didn't work](https://github.com/Ayesh/php-src/runs/7427009578?check_suite_focus=true#step:9:4050), even though it was tested on CGI. I think the way phpinfo determines whether to use HTML output or CLI output has to do with this.

Reply:
Ugh, an empty `--CGI--` section is no longer recognized by run-tests.php as of 9140c9038a83ff55a78f357f8485de086d83d94e (PHP 8.1.0). [This should be fixed](https://github.com/php/php-src/pull/9061).

Reply:
Fantastic, I tested with the change in that PR, and the test indeed passed with `--CGI--`. I will rebase and push to this PR after #9061 is merged. Thank you.

Reply:
Rebased, thank you.